composer.json

```diff
@@ -58,6 +58,7 @@
         "drupal/moderation_dashboard": "1.0.0-beta3",
         "drupal/moderation_sidebar": "^1.4",
         "drupal/mysql56": "^1.0",
+        "drupal/node_revision_delete": "^1.0@RC",
```

Comment:
@rajeshreeputra Why are we including the module here as we've already included in acquia_cms_site_studio module ?

Reply:
removed node_revision_delete requirement from acquia_cms/composer.json and pushed changes.